index.html

```diff
@@ -5554,7 +5554,7 @@ <h3 id='example'>Example</h3>
 
 <p>&nbsp;</p>
 <h1 id='claim'>Claim</h1><h2 id='claim-2'>Claim</h2><h3 id='overview'>Overview</h3>
-<p>A list of services and products provided, or to be provided, to a patient.</p>
+<p>List of services and products billed to a patient or insurance company.</p>
```

Comment:
Aren't these things only ever billed to insurance companies? We shouldn't mention anything about patient.

Reply:
Fixed.